mautic/mautic
# Fix warning from last build and  remove icon on unpublished segment

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

PR disable warning icon for unpublished segment
Also fix counting of inactive hours



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  You can directly in DB change in table lead_lists change for one of the segment last_build_date older like 30 days
3. See like there is no warning icon like we know from Mautic5
4. After PR you should see it
5. Unpublish segment
6. You should not see warning icon
